### ISSUE TITLE
Make sure website works without javascript

### ISSUE THREAD

**Issue by lw64** [vala-lang/vala-docs#52]
I haven't checked that yet, it would be nice if someone who cares about this can do! :)

**Comment by lw64** [comment]
also accessibility features like screen readers should work

**Comment by colinkiama** [comment]
> I haven't checked that yet, it would be nice if someone who cares about this can do! :)

It works fine. You lose out on the following:
- Search
- Copy Button in code blocks

**Comment by colinkiama** [comment]
> also accessibility features like screen readers should work

From reading the HTML alone, they should work, Screen Readers work. The HTML generated is structured and semantic. However, accessibility should be discussed in separate issues. However I think this should be part of its own issue. Where somebody can specifically verify this.
